scripts/itkdev-docker-compose

```diff
@@ -225,6 +229,19 @@ EOF
     echo "0.0.0.0 $DNS # itkdev-docker-compose" | sudo tee -a /etc/hosts
     ;;
 
+  hosts:add-host-ip)
+    DNS=${COMPOSE_PROJECT_NAME}.docker.localhost
+    # Remove any existing entry (note: `sed --in-place` will not work on
```

Comment:
What existing entry are you removing?

Reply:
I'm preventing adding multiply entries for the host ip, i.e. cleaning up when the `hosts:add-host-ip` command is run multiple times.